build/yamls/flow-visibility/base/provisioning/dashboards/pod_to_pod_dashboard.json

```diff
@@ -566,7 +559,7 @@
           },
           "format": 1,
           "queryType": "sql",
-          "rawSql": "select SUM(octetDeltaCount) as bytes, sourcePodNamespace\nFrom flows_pod_view\nWHERE flowType IN (1, 2)\nAND sourcePodNamespace NOT IN ('kube-system', 'flow-visibility', 'flow-aggregator')\nAND destinationPodNamespace NOT IN ('kube-system', 'flow-visibility', 'flow-aggregator')\nAND $__timeFilter(flowEndSeconds)\nGROUP BY sourcePodNamespace\nHAVING bytes > 0",
+          "rawSql": "select SUM(octetDeltaCount) as bytes, sourcePodNamespace\nFrom flows_pod_view\nWHERE flowType IN (1, 2)\nAND sourcePodNamespace NOT IN ('kube-system', 'flow-visibility', 'flow-aggregator')\nAND destinationPodNamespace NOT IN ('kube-system', 'flow-visibility', 'flow-aggregator')\nAND $__timeFilter(flowEndSeconds)\nGROUP BY sourcePodNamespace\nHAVING bytes > 0\nORDER BY bytes DESC",
```

Comment:
I'm curious about why there is no limit in this query? Is it because this query only return one row?

Reply:
good catch! The limit is added in L547. The pie chart panel provides such a configuration parameter  - "limit":  https://grafana.com/docs/grafana/latest/visualizations/pie-chart-panel/#limit
so that we don't need to add it in the query